frontend/device/generic/device.lua

```diff
@@ -410,7 +410,7 @@ function Device:reboot() end
 -- Hardware specific method to initialize network manager module
 function Device:initNetworkManager() end
 
-function Device:supportsScreensaver() return false end
+function Device:supportsScreensaver() return true end
```

Comment:
Unintended leftover from the Emu/SDL shenanigans?

Reply:
You can pretty much do the same for the emulator with

```lua
function Emulator:supportsScreensaver() return true end
```

in https://github.com/koreader/koreader/blob/master/frontend/device/sdl/device.lua#L338

Reply:
Yes, I will remove this since it's not necessary for the PR.